asset/quickstart.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// Imports the Google APIs client library
+async function exportAssets (dumpFilePath) {
+  // [START asset_quickstart_exportassets]
+  const asset = require('@google-cloud/asset');
+  var client = new asset.v1beta1.AssetServiceClient({
+    // optional auth parameters.
+  });
+
+  // Your Google Cloud Platform project ID
+  const projectId = process.env.GCLOUD_PROJECT;
+  var projectResource = client.projectPath(projectId);
+
+  // var dumpFilePath = 'Dump file path, e.g.: gs://<my_bucket>/<my_asset_file>'
+  var outputConfig = {
+    gcsDestination: {
+      uri: dumpFilePath
+    }
+  };
+  var request = {
+    parent: projectResource,
+    outputConfig: outputConfig
+  };
+
+  // Handle the operation using the promise pattern.
+  client.exportAssets(request).then(responses => {
```

Comment:
Since you have an `async` function, you could use `await` instead of promises, which is much nicer: 

```
let responses = await client.exportAssets(request);
var operation = responses[0];
responses = await operation.promise();
var result = responses[0];
```

Feel free to leave out the `catch` in the sample. If the sample throws, we'll see the error either way.